core/src/main/kotlin/org/eln2/parsers/falstad/Falstad.kt

```diff
@@ -130,15 +133,14 @@ class Falstad(val source: String) {
 		if(DEBUG) {
 			val repmap: MutableMap<PosSet, MutableSet<PosSet>> = mutableMapOf()
 			roots.values.forEach {
-				dprintln("F.<init>: r $it => ${it.representative}")
+				dprintln("F.<init>: r ${it} => ${it.representative}")
```

Comment:
need to revert this change

Reply:
The change here involves adding braces to a format string; this actually triggers a lint in IDEA ("Remove unnecessary braces"). Are you sure about this one?

Also, it's in the debug path, not at all hotpath or maintainability target.

Reply:
The git changes seem to be reverting the change, the braces are not needed (as IDEA says)

Reply:
I'm dumb, I read the diff in the wrong order :)